apps/cowswap-frontend/src/modules/volumeFee/state/cowswapFeeAtom.ts

```diff
@@ -0,0 +1,42 @@
+import { atom } from 'jotai'
+
+import { GNOSIS_CHAIN_STABLECOINS, NATIVE_CURRENCY_ADDRESS } from '@cowprotocol/common-const'
+import { getIsNativeToken, isInjectedWidget } from '@cowprotocol/common-utils'
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+import { walletInfoAtom } from '@cowprotocol/wallet'
+
+import { derivedTradeStateAtom } from 'modules/trade'
+
+import { VolumeFee } from '../types'
+
+const COWSWAP_VOLUME_FEE: VolumeFee = {
+  bps: 10, // 0.1%
+  recipient: '0x6b3214fD11dc91De14718DeE98Ef59bCbFcfB432', // Gnosis Chain Protocol fee safe
```

Comment:
I know that PR comes after this one, but should afterwards change this to use the chain map, to make sure it's included only on gnosis chain based on the configuration, and make the logic simpler?

Reply:
I love this suggestion, thanks!
https://github.com/cowprotocol/cowswap/pull/4547/commits/d6dd53dba7feffe422cd3d2fd9382305679dd2d9